packages/flutter/lib/src/gestures/tap_and_drag.dart

```diff
@@ -684,6 +684,13 @@ mixin _TapStatusTrackerMixin on OneSequenceGestureRecognizer {
 /// pointer does travel enough distance then the recognizer that entered the arena
 /// first will win. The gesture detected in this case is a drag.
 ///
+/// {@tool dartpad}
+/// This example shows how to use the TapAndPanGestureRecognizer along with a
+/// RawGestureDetector to scale a Widget.
```

Comment:
Should the two classes mentioned here have square brackets around them?

Reply:
These should have brackets.